### DISCUSSION TITLE
Calculating index values

### DISCUSSION THREAD

**Discussion by Braulio-Tapia** [sdmTMB/sdmTMB#246] [Q&A]
Dear group, 
I am implementing a spatial model for estimating the mean density of rainbow trout in a lake using hydroacoustic echo counts (horizontal transducer setup). My observations consist of number of trouts per insonified volume of water (m3). Using design-based estimators, we obtain mean density of 0.000458 fish/m3 for the entire lake.
Now, using sdmTMB with spatial random effects, I obtain a mean density estimate of 0.13. This is roughly three orders of magnitude larger than the previous estimate.
Inputs to the model
1-. The density is provided spatially in a regular grid.
2-. The grid was developed from X and Y (UTM) in km units.
3-. Cells of grid were only delimited for aquatic ecosystem based on a lake coastline with _add_barrier_mesh_
4-. Based on different tests of probability distributions, i.e., tweedie, delta-gamma, delta-lognormal, delta-poisson-link-gamma, delta-poisson-link-lognormal; I selected delta-lognormal distribution
5-. Then, the model configuration: 
  _sdmTMB(density ~ -1 + as.factor(stratum), mesh= bspde, time= "year", 
                spatiotemporal= list("off","off"), spatial= list("on","on"), anisotropy= F,
                family= delta_lognormal (link1="logit", link2="log"), data= data)_ 

Any feedback that helps me track this tremendous difference would be helpful and greatly appreciated 

Thanks, Braulio.

**Comment by seananderson** [comment]
I've converted this to a 'Discussions' question from the issue tracker.

How exactly are you comparing the two? The output from `get_index()` will be in units of fish (or biomass in the case of biomass density), not density. This does the area expansion from density to the entire prediction grid. Are you trying to expand over all depths given this is per m^3?

Can you provide a complete example even with fake or simplified data? You can email me the code + data if needed and I can follow up here.

If your data are counts, you might consider a negative binomial or similar, but maybe your data are actually density here.

Previous work has shown the lognormal can be biased a bit high compared to a design-based estimators, but not that much.

**Comment by Braulio-Tapia** [comment]
Thanks Sean, I´ve sent you an email
Braulio
